spec/rubocop/cop/performance/array_push_single_spec.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Performance::ArrayPushSingle, :config do
+  it 'registers an offense and corrects when using `push` with a single element' do
+    expect_offense(<<~RUBY)
+      array.push(element)
+      ^^^^^^^^^^^^^^^^^^^ Use `<<` instead of `push`.
+    RUBY
+
+    expect_correction(<<~RUBY)
+      array << element
+    RUBY
+  end
+
+  it 'registers an offense and corrects when using `push` with a single element and safe navigation operator' do
+    expect_offense(<<~RUBY)
+      array&.push(element)
+      ^^^^^^^^^^^^^^^^^^^^ Use `<<` instead of `push`.
+    RUBY
+
+    # gross. TODO: make a configuration option?
```

Comment:
What's the best way to handle this?

Reply:
I'd just ignore it. `&.<<` looks horrible

Reply:
Agree.